components/README.md

```diff
@@ -0,0 +1,14 @@
+# Atmos Components
```

Comment:
With Atmos you basically configure your "stack" with several "components" (Helm, TF). I added few Readme files so the purpose of each folder is clear.